src/page-weekly-snapshot/LocaleStatsTable.tsx

```diff
@@ -0,0 +1,329 @@
+  numFacilitiesWithRtGreaterThan1PrevWeek,
+  numFacilitiesWithRtGreaterThan1ThisWeek,
+} from "../page-response-impact/rtStatistics";
+import { NYTCountyRecord, NYTStateRecord, useNYTData } from "./NYTDataProvider";
+import {
+  BorderDiv,
+  COLUMN_SPACING,
+  Heading,
+  HorizontalRule,
+  Left,
+  LeftHeading,
+  RankContainer,
+  RankText,
+  Right,
+  Table,
+  TableHeading,
+  TextContainer,
+  TextContainerHeading,
+} from "./shared";
+import { useWeeklyReport } from "./weekly-report-context";
+
+type PerCapitaCountyCase = {
+  name: string;
+  casesIncreasePerCapita: number | undefined;
+};
+
+function getDirection(
+  numFacilitiesLastWeek: number,
+  numFacilitiesThisWeek: number,
+) {
+  return numFacilitiesLastWeek <= numFacilitiesThisWeek ? "+" : "-";
+}
+
+function getDay(nytData: NYTCountyRecord[] | NYTStateRecord[], day: number) {
+  if (day === 1) {
+    return minBy(nytData, (d: NYTCountyRecord | NYTStateRecord) => d.date);
+  } else {
+    return maxBy(nytData, (d: NYTCountyRecord | NYTStateRecord) => d.date);
+  }
+}
+
+function getCountyDataByName(counties: NYTCountyRecord[]) {
+  const dataByCounty: { [countyName: string]: NYTCountyRecord[] } = {};
+
+  counties.forEach((data: NYTCountyRecord) => {
+    if (dataByCounty[data.county]) {
+      dataByCounty[data.county] = [...dataByCounty[data.county], data];
+    } else {
+      dataByCounty[data.county] = [data];
+    }
+  });
+
+  return dataByCounty;
+}
+
+function calculatePerCapitaIncrease(
+  daySevenCases: number | undefined,
+  dayOneCases: number | undefined,
+  countyPopulation: number | undefined,
+) {
+  if (
+    (!daySevenCases && daySevenCases !== 0) ||
+    (!dayOneCases && dayOneCases !== 0) ||
+    !countyPopulation
+  )
+    return;
+  return (daySevenCases - dayOneCases) / countyPopulation;
+}
+
+function getCountyIncreasePerCapita(
+  counties: NYTCountyRecord[],
+  stateLocaleData: Map<string, LocaleRecord> | undefined,
+) {
+  const dataByCounty = getCountyDataByName(counties);
+  const countyCasesIncreasePerCapita: PerCapitaCountyCase[] = [];
+
+  for (const countyName in dataByCounty) {
+    const dayOne = getDay(dataByCounty[countyName], 1);
+    const daySeven = getDay(dataByCounty[countyName], 7);
+    const countyPopulation = stateLocaleData?.get(countyName)?.totalPopulation;
+    countyCasesIncreasePerCapita.push({
+      name: countyName,
+      casesIncreasePerCapita: calculatePerCapitaIncrease(
+        daySeven?.cases,
+        dayOne?.cases,
+        countyPopulation,
+      ),
+    });
+  }
+  return countyCasesIncreasePerCapita;
+}
+
+function getFacilitiesInCountiesToWatch(
+  facilities: Facility[],
+  countiesToWatch: string[],
+) {
+  let result = "";
+  for (let i = 0; i < facilities.length; i++) {
+    const modelInputs = facilities[i].modelInputs;
+    const facilityCounties = Object.values(pick(modelInputs, "countyName"));
+    for (let j = 0; j < facilityCounties.length; j++) {
+      const currCounty = facilityCounties[j];
```

Comment:
there can't ever be more than one item in `facilityCounties`, can there? if not, it might be more straightforward to just say `currCounty = modelInputs.countyName`

Reply:
There is only one county per facility! My mistake, thanks for catching this.